bril-txt/briltxt.py

```diff
@@ -210,6 +222,12 @@ def float(self, items):
     def nullptr(self, items):
         return 0
 
+    def char(self, items):
+        value = str(items[0])[1:-1] # Strip `'`.
+        if value in control_chars.keys():
```

Comment:
```suggestion
        if value in control_chars:
```

Just doing `key in dict` suffices to check for key existence.